test/controllers/update/update.beneficial.owner.other.controller.spec.ts

```diff
@@ -113,7 +112,6 @@ describe("UPDATE BENEFICIAL OWNER OTHER controller", () => {
   describe("GET tests", () => {
     test(`Renders the ${UPDATE_BENEFICIAL_OWNER_OTHER_PAGE} page`, async () => {
       mockGetApplicationData.mockReturnValueOnce({ ...APPLICATION_DATA_UPDATE_BO_MOCK });
-
       const resp = await request(app).get(UPDATE_BENEFICIAL_OWNER_OTHER_URL);
 
       expect(resp.status).toEqual(200);
```

Comment:
do you want to undo your changes in this file?

Reply:
Done